spec/models/shipping_rate_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+require 'shipping_rate'
+
+RSpec.describe ShippingRate do
+  let(:location1) { ShippingLocation.new(country: 'US', state: 'CA', city: 'Beverly Hills', zip: '90210') }
+  let(:location2) { ShippingLocation.new(country: 'US', state: 'WA', city: 'Seattle', zip: '98101') }
+  let(:package) { ActiveShipping::Package.new(12, [15, 10, 4.5], :units => :imperial) }
+
+  describe 'validations' do
+    it 'is valid' do
+      r = ShippingRate.new(origin: location1, destination: location2, package: package)
```

Comment:
I'm not a big fan of single-letter variables as they don't explain what they are. Especially reading line 13 as english it's not clear what you are testing.